Praktikum/Tag02/uploads/01_strings_verbinden/strings_verbinden.c

```diff
@@ -2,18 +2,17 @@
 #include "strings_verbinden.h"
 
 void strings_verbinden(char *zusammen, char *string1, char *string2) {
-  int pos_string1 = 0, pos_zusammen = 0;
+  int pos_zusammen = 0;
 
   // Insert the first string in the new string
-  while (string1[pos_string1] != '\0') {
+  for (int pos_string1 = 0; string1[pos_string1] != '\0';) {
```

Comment:
`i` bzw `pos_string1` wird nur innerhalb der while-Schleife gebraucht. Es empfiehlt sich Variablen auf den Bereich zu beschränken, wo sie benutzt werden um versehentliche Verwendung (weil man vergessen/übersehen hat, dass die Variable schon in diesem Bereich definiert ist) oder Überschattung (man definiert die Variable unwissentlich erneut und "überdefiniert" damit eine andere Stelle versehentlich) zu vermeiden. Eine for-Schleife ist vielleicht nicht so hübsch wie eine while-Schleife, aber pos_string1 ist nur innerhalb dieser definiert.